model_analyzer/config/input/config_command_report.py

```diff
@@ -121,11 +121,14 @@ def _fill_config(self):
                     'Comma delimited list of the names of model configs'
                     ' for which to generate detailed reports.')))
         self._add_config(
-            ConfigField('output_format',
-                        flags=['--output-format', '-o'],
+            ConfigField('output_formats',
+                        flags=['--output-formats', '-o'],
                         default_value=DEFAULT_REPORT_FORMAT,
-                        field_type=ConfigListGeneric(type_=ConfigEnum(
-                            choices=['pdf', 'csv', 'png'])),
+                        field_type=ConfigUnion([
+                            ConfigListGeneric(type_=ConfigEnum(
+                                choices=['pdf', 'csv', 'png'])),
+                            ConfigListString()
```

Comment:
What is the case not covered by `ConfigListGeneric`?

Reply:
It can't seem to handle comma separated list of strings like `pdf,csv,svg`